### ISSUE TITLE
CORR-001: corrosion/coating formulated-material discovery and degradation benchmark

### ISSUE THREAD

**Issue by Tristan-Stoltz-ERC** [Luminous-Dynamics/symthaea#5255]
## Mission

Create a materials-discovery campaign for corrosion protection that treats a protective coating or surface system as an exact **formulated material + substrate + surface preparation + application/process state + exposure history**, not as a timeless scalar `corrosion_resistance` property.

CORR-001 is intended to be one of the first accessible physical-validation demonstrations for the broader MAT-HYPOTHESIS / MAT-OPPORTUNITY / MAT-014 architecture.

## Why this domain is useful

A 2026 proof-of-principle study used a relatively sparse historical coating dataset (492 salt-spray results spanning 148 ingredients), generated novel ingredient combinations with machine learning, and physically tested two predicted formulations under cyclic corrosion conditions. The reported formulations were similar to or better than the reference in the tested profiles, while the authors explicitly noted that the data-driven result did not itself explain the mechanism and that some feature effects were unexpected.

Reference:
- https://www.nature.com/articles/s41529-026-00771-2

A separate 2026 corrosion perspective argues that corrosion is a trajectory rather than one static event and motivates dynamic/world-model-style representations for predicting response to interventions:
- https://www.nature.com/articles/s41529-026-00826-4

These are external motivation and benchmark-design inputs, not local reproduction evidence.

## Governing separations

```text
coating formulation
!= cured coating state

cured coating state
!= coating/substrate interface state

initial performance
!= durability

salt-spray score
!= universal corrosion resistance

visual rust/blister rating
!= electrochemical mechanism

model feature importance
!= causal mechanism

high predicted protection
!= physically validated protection

one accelerated exposure
!= field lifetime

corrosion protection
!= commercial coating viability
```

## Formulated-material subject

Reuse MAT-007 where possible, but make the formulation/process extension explicit.

Bind at minimum where applicable:

- resin/binder identities;
- pigments/fillers/inhibitors/additives;
- solvent/water/vehicle state;
- exact component fractions and normalization basis;
- component source/grade where material;
- mixing/order/process profile;
- application method;
- wet/dry film thickness;
- coat count/layer stack;
- cure temperature/time/humidity profile;
- substrate material/alloy/condition;
- substrate surface preparation;
- primer/conversion layer where present;
- deliberate defect/scribe geometry where used;
- coating age before exposure;
- specimen geometry/area;
- deterministic subject identity.

A change in cure, film thickness, substrate preparation or layer stack is a scientific subject/state change, not metadata trivia.

## Exposure subject

Create an exact environmental/exposure profile rather than the label `salt spray`.

Bind where applicable:

- exposure protocol/standard revision;
- wet/dry/immersion cycle structure;
- solution species/concentrations;
- pH;
- temperature;
- relative humidity;
- gas/oxygen state where relevant;
- flow/spray/deposition conditions;
- UV exposure where relevant;
- mechanical damage/scribe state;
- applied electrical/electrochemical condition where relevant;
- total elapsed exposure;
- sampling/inspection intervals;
- specimen orientation;
- chamber/site identity and calibration evidence for physical tests.

```text
same nominal coating
+ different exposure profile
!= same corrosion proposition
```

## Time-conditioned degradation evidence

Represent corrosion performance as a trajectory where data exist.

Potential observations include:

- rusted/corroded area;
- blistering;
- scribe creep / underfilm corrosion;
- mass change;
- coating thickness change;
- adhesion change;
- cracking/delamination;
- water uptake/permeation proxies;
- open-circuit potential;
- polarization/electrochemical observations;
- EIS-derived observations under exact analysis profile;
- surface/cross-section characterization;
- image observations;
- failure/onset time under declared criterion.

Do not collapse these into one universal corrosion score.

## Observer / rating authority

Many corrosion/coating outcomes use ordinal or human-interpreted visual ratings.

Bind:

- exact rating standard/profile;
- observer/evaluator identity/class;
- image/specimen evidence refs;
- inter-rater or automated-rating calibration where available;
- uncertainty/ambiguity;
- exact time point.

Preserve:

```text
visual rating
!= direct physical measurement
```

and:

```text
one automated image model
!= independent observer
```

## Mechanism hypotheses

Possible mechanism classes include, without asserting any one is correct:

```text
BarrierTransport
ChargeTransferSuppression
ActiveInhibitorRelease
SacrificialProtection
Passivation
AdhesionRetention
WaterOrIonUptake
DefectHealing
InterfacialChemistry
Multimechanism
Unknown
```

Use MAT-HYPOTHESIS #5254 to represent exact proposals and competing explanations.

Example:

```text
formulation performs better
```

does not establish whether the cause is inhibitor chemistry, resin synergy, lower permeability, improved adhesion, altered cure morphology, thickness, or another covarying factor.

Mechanism-discriminating experiments should be separately proposed through RQ-043 / MAT-014.

## Causal bottleneck bridge

Use MAT-OPPORTUNITY-003 #5245 to distinguish material/formulation limitation from other causes of field corrosion such as:

- poor surface preparation;
- geometric crevices/water traps;
- coating damage;
- galvanic couples;
- drainage/ventilation;
- cathodic protection state;
- maintenance interval;
- operating environment.

A better coating may have little system value when another failure mode dominates.

## Candidate search space

Allow several intervention classes:

- formulation ingredient selection/fractions;
- resin/binder combinations;
- inhibitor/pigment/filler combinations;
- multilayer/primer/topcoat architecture;
- film thickness;
- cure/process changes;
- surface treatment/interface changes;
- self-healing/active-release systems where evidence permits;
- substrate/coating pair selection;
- non-material/process/maintenance alternatives.

Formulation feasibility constraints belong in the candidate-generation profile rather than being learned implicitly from only successful historical recipes.

## Dataset bias / missing negatives

Coating development datasets are often strongly selected toward formulations worth testing.

Track:

- failed/unmixable/uncoatable formulations;
- cure failures;
- adhesion failures;
- poor appearance/processability;
- early corrosion failures;
- missing tests;
- censored test durations;
- recipe components excluded by human craft knowledge;
- duplicated/near-duplicated formulations;
- formulation families from the same development lineage.

Random row splits can leak formulation-family/process knowledge. Use formulation-family and temporal/source holdouts where possible.

## Retrospective benchmark

Do **not** claim reproduction of the 2026 AkzoNobel study from its headline metrics: the underlying training/validation data are proprietary according to the paper.

Instead:

1. use that work as a methodology/benchmark-design reference;
2. assemble a separately licensed/public/open retrospective corpus where possible;
3. freeze exact data/source rights and conditions;
4. include negative/inconclusive records;
5. compare simple baselines before sophisticated models;
6. preserve formulation-family/source/time holdouts;
7. report the limits of any small heterogeneous corpus.

## First prospective physical benchmark

After the evidence/authority layers are qualified, prefer a deliberately narrow coupon experiment rather than immediately attempting a commercial coating search.

Target structure:

```text
small frozen formulation set
+ one exact substrate/preparation profile
+ one exact application/cure profile
+ one declared accelerated exposure protocol
+ repeated specimens / controls
+ blinded or independently reconstructed ratings where possible
+ preregistered analysis
```

The first goal is to test the **research loop**, not to claim a market-ready coating.

Choose conditions/materials only through an explicit lab safety/protocol review. CORR-001 grants no chemical handling or automated experiment authority.

## Useful low-cost scientific questions

The first benchmark should favor hypotheses that can fail decisively, e.g.:

- does a predicted formulation ranking survive physical exposure?
- does a formulation-family holdout reveal model overfitting?
- does a claimed ingredient effect survive controlled formulation context?
- does early image/electrochemical evidence predict later degradation under the exact profile?
- can the model identify when it is outside its formulation/application domain?
- does a simple baseline match the sophisticated search method?

## Dynamic/world-model extension

Later, support state-transition hypotheses such as:

```text
coating state_t
+ environment/intervention_t
-> distribution over coating/interface state_{t+1}
```

but preserve:

```text
predicted degradation trajectory
!= observed trajectory
```

and score only predictions committed before later inspection evidence.

This can become an unusually good testbed for Symthaea's broader predictive-model arbitration/currentness machinery because degradation is sequential and interventions may change the trajectory.

## Multi-objective campaign

Preserve a Pareto view over independently measured/estimated quantities such as:

- corrosion protection under exact exposure;
- durability/time to failure;
- adhesion/mechanical integrity;
- processability/application window;
- cure burden;
- coating thickness/material intensity;
- environmental/health constraints where evidence exists;
- critical-material burden;
- cost under dated scenario;
- repair/recoatability;
- uncertainty/calibration.

No universal `coating score`.

## Hidden-discovery / hypothesis benchmark

CORR is a good MAT-HYPOTHESIS domain because non-intuitive formulation synergies can exist.

For historical tests:

```text
pre-cutoff formulation/evidence corpus
-> frozen hypotheses/formulation proposals
-> held-out later evidence
```

with exact exposure-control semantics from #5254.

Modern open-world LLM exposure must not be presented as leakage-free historical rediscovery.

## Baselines

At minimum compare where data permit:

```text
random feasible formulation
nearest-known formulation
simple ingredient-frequency / linear baseline
tree/boosting baseline
uncertainty-aware active-learning baseline
more sophisticated generative/LLM/Symthaea proposal
```

and report when the simple baseline wins.

## Adversarial fixtures

1. same ingredients/fractions + different cure -> distinct subject;
2. same coating + different substrate prep -> distinct interface state;
3. same coating + different dry-film thickness -> distinct proposition;
4. same numerical rating under different exposure protocols -> not directly interchangeable;
5. great initial result + rapid later blistering -> durability tradeoff remains visible;
6. feature importance changes under formulation-family holdout -> no stable causal claim;
7. correlated ingredient receives high model importance due formulation convention -> no mechanism promotion;
8. formulation suggested by model cannot be practically mixed/applied -> candidate feasibility failure preserved;
9. high visual score + poor adhesion -> no universal coating superiority;
10. same specimen image scored differently by two observers -> disagreement retained;
11. post-outcome model revision cannot receive prospective credit for that outcome;
12. accelerated test success cannot be relabeled field-lifetime evidence;
13. model trained on one substrate/exposure cannot silently generalize to another;
14. physical null/contradictory result updates hypothesis evidence but does not erase original prediction;
15. proprietary-paper headline result cannot become local reproduced evidence without underlying data/execution.

### `CORR-001A` — formulated coating / substrate / process / exposure identity

Pure deterministic scientific contracts.

### `CORR-001B` — time-conditioned degradation observations and rating profiles

Preserve raw measurements/images/ordinal ratings and their distinct authority classes.

### `CORR-001C` — retrospective open-data benchmark

Freeze a licensed corpus, family/time/source splits and simple baselines.

### `CORR-001D` — hypothesis/mechanism bridge

Compose MAT-HYPOTHESIS #5254 and causal bottleneck work #5245.

### `CORR-001E` — prospective coupon protocol and evidence custody

Protocol/analysis preregistration only; no automatic physical execution.

### `CORR-001F` — independently authorized physical pilot

Only after MAT-013/MAT-014 and relevant lab safety/authority paths are ready.

### `CORR-001G` — degradation trajectory / model arbitration

Prospective sequential predictions and predicted-vs-observed trajectory calibration.

## Dependencies / relationships

- MAT-007 #4336 material/architecture identity
- MAT-008 #4339 conditioned evidence
- MAT-009 #4340 Pareto campaigns
- MAT-010 #4341 negative/search memory
- MAT-011 #4376 calibrated multi-fidelity evaluation
- MAT-012 #4377 novelty/synthesizability separation
- MAT-013 #4378 physical sample/process lineage
- MAT-014 #4345 human-authorized active discovery
- MAT-016 #4991 advisory/evidence firewall
- MAT-OPPORTUNITY-003 #5245
- MAT-HYPOTHESIS-001 #5254
- MAT-PORTFOLIO-001 #5247
- RQ-043 #2579

Do not stack implementation on currently unqualified MAT-OPPORTUNITY source #5212.

## Claim ceiling

CORR-001 may establish exact formulated-material, process, exposure and degradation evidence and support bounded prospective comparisons under those conditions. It does not establish universal corrosion resistance, causal mechanism from prediction alone, field lifetime from accelerated exposure, regulatory/environmental suitability, commercial coating viability, experiment authority, or product qualification.
